iModelCore/ECDb/Tests/NonPublished/ECSqlStatementTests.cpp

```diff
@@ -1415,6 +1415,51 @@ TEST_F(ECSqlStatementTestFixture, IsNull)
             }
         }
 
+        //*** array with two null elements and insert with only

```

Comment:
Is it worth duplicating all these tests that focus on something besides insert to test only?  I don't want to disincentivize testing but do these test cases add any thing special not covered in the more insert focused tests at the bottom of this file?